app/webpacker/styles/application.scss

```diff
@@ -30,12 +30,20 @@ a[href="#"] {
 
 .trainee-record-header {
   margin-bottom: govuk-spacing(6);
-}
 
-.trainee-record-header .govuk-tag {
-  @include govuk-media-query($from: tablet) {
+  &__name-trn {
```

Comment:
I'd avoid creating your selector like this - the design system never uses it because it isn't then searchable.